### DISCUSSION TITLE
Ability to remove "Welcome to Ray!" message + instructions

### DISCUSSION THREAD

**Discussion by aryehraber** [spatie/ray#465] [Ideas]
Firstly, thank you for creating Ray, it is SO useful and much better than `dump`ing all over the place!

Small FR: It would be great to have a config option to remove the welcome message + instructions from blank screens. While it's definitely useful to have this text appear on first open, I don't really want to see it every time I clear the screen (which I do often), I'd prefer see a fully blank screen until the next message(s) come in.

**Comment by freekmurze** [comment]
Thanks for the kind words on Ray. We'll consider adding this in the near future.

**Comment by Brugman** [comment]
The blank screen message has changed but I too would prefer a blank blank screen.

**Comment by nickturrietta** [comment]
Please consider adding this as a feature, or let us override it. The Welcome screen is helpful for onboarding, but after onboarding is just noisy and clutters my screen and mindspace.

**Reply by freekmurze** [reply]
We're working on v3 of Ray. This screen wil be totally revamped.
